src/index.ts

```diff
@@ -39,7 +40,7 @@ const isInlineStyle = pred.AND(
 /**
  * Transforms all inline styles in `html` with `filter`
  */
-function html(text) {
+export function html(text: string) {
```

Comment:
For legibility we've been generally following the practice of explicitly annotating the return types of exported functions